apps/docs/playwright.config.js

```diff
@@ -0,0 +1,106 @@
+// @ts-check
+const { devices } = require("@playwright/test");
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * @see https://playwright.dev/docs/test-configuration
+ * @type {import('@playwright/test').PlaywrightTestConfig}
+ */
+const config = {
+  testDir: "./e2e",
```

Comment:
Lets have tests in the same folder as sources (similar to jest but Playwright tests will be `.spec.js`, mostly for pages, and Jest tests will be `.test.js`, mostly for components)